app/screens/interactive_dialog/interactive_dialog.js

```diff
@@ -123,17 +127,29 @@ export default class InteractiveDialog extends PureComponent {
 
         this.submitted = true;
 
-        if (!data || !data.errors || Object.keys(data.errors).length === 0) {
-            this.handleHide();
-            return;
+        let hasErrors = false;
+
+        if (data) {
+            if (data.errors &&
+                Object.keys(data.errors).length >= 0 &&
+                checkIfErrorsMatchElements(data.errors, elements)
+            ) {
+                hasErrors = true;
+                this.setState({errors: data.errors});
```

Comment:
I'm not seeing `this.state.errors` being used anywhere.

Also please add a test case for multiple errors, if that is a use case

Reply:
@enahum `this.state.errors` is used for error messages on each specific input field.  This feature was already there prior to this PR. 

It is missing tests, but perhaps that should be done in another PR.  What do you think?